app/src/components/CompetitionHeader.tsx

```diff
@@ -5,33 +5,39 @@ export const CompetitionHeader = ({
   name,
   registrationOpen,
   registrationClose,
-  series,
+  isRegistrationDifferent = false,
```

Comment:
I found this parameter name unclear. I think `doSeriesRegistrationsDiffer` or something would be better (assuming my understanding is correct).